package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "@eclipse-cdt-cloud/vscode-ui-components",
+    "displayName": "VSCode UI Components",
+    "description": "VSCode UI Components for React",
+    "version": "0.0.1",
+    "publisher": "eclipse-cdt",
+    "license": "MIT",
+    "repository": "https://github.com/eclipse-cdt-cloud/vscode-ui-components",
+    "qna": "https://github.com/eclipse-cdt-cloud/vscode-ui-components/issues",
+    "main": "./lib/index.js",
+    "types": "./lib/index.d.ts",
+    "files": [
+        "lib"
+    ],
+    "engines": {
+        "node": ">=20"
+    },
+    "scripts": {
+        "prepare": "yarn clean && yarn build",
+        "clean": "rimraf ./lib ./node_modules/.tmp",
+        "build": "tsc && yarn lint",
+        "watch": "tsc -w",
+        "lint": "eslint ."
+    },
+    "dependencies": {
+        "@floating-ui/react": "^0.26.17",
```

Comment:
Quite a few dependencies. We need to think about adding TPIP support documentation.

Reply:
How would that look?

Reply:
On some none Eclipse CDT Cloud OSS projects we have something like this:
https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/blob/main/TPIP.md
which is generated by some scripts from an MD file and a JSON file in this folder
https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/tree/main/docs

But thinking about it again, we may want to explore first if we have already approaches in the CDT cloud organization. And if we should agree on a single approach for the entire organization in our monthly calls. @planger , maybe one to add to the next monthly call's agenda.